notebooks/writing_a_trace_transform_cpu_offloading.ipynb

```diff
@@ -0,0 +1,752 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    offload_exec = OperatorExecutor("offload_exec")
</pre><p>offload_ex, for sort of consistency with the existing executors?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Lightning-AI/lightning-thunder/pull/966/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>